components/widgets/admin_console/admin_user_card.jsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {Client4} from 'mattermost-redux/client';
+
+import ProfilePicture from 'components/profile_picture.jsx';
+
+const WrapperDiv = styled.div`
```

Comment:
Why are we using `styled-components`? I thought we were moving towards using CSS modules instead?

Reply:
There are a few reasons I'd like to discuss in person, but essentially CSS modules just wasn't working with our current webapp build. I was also struggling to integrate it with the existing CSS classes.